python/cloud-init/scripts/kube-vip-lb-services.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+echo "Install kube-vip for Load Balancing..."
+
+IMAGE=ghcr.io/kube-vip/kube-vip:v0.6.0
```

Comment:
perhaps:
```suggestion
IMAGE=${IMAGE:-ghcr.io/kube-vip/kube-vip:v0.6.0}
```

Reply:
and then perhaps with a better name matching `KUBE_VIP_IMAGE`?

```sh
KUBE_VIP_VERSION=${KUBE_VIP_VERSION:-v0.6.0}
KUBE_VIP_IMAGE=${KUBE_VIP_IMAGE:-ghcr.io/kube-vip/kube-vip:$KUBE_VIP_VERSION}
```

Reply:
addressed in https://github.com/equinix-labs/pulumi-equinix-kubernetes-cluster/pull/9/commits/a0b9506847dc23d818987ed5bc592684addac079